### ISSUE TITLE
multi projects with shared folder

### ISSUE THREAD

**Issue by mikimoresi** [shuidi-fed/vapper#52]
Hello,
I am trying to create a multiproject folder like described here [https://dev.to/perpetualwar/sharing-component-logic-between-vue-apps-1fg](https://dev.to/perpetualwar/sharing-component-logic-between-vue-apps-1fg)
I have created a repo to test my issue: [https://github.com/mikimoresi/vapper_multi_projects](https://github.com/mikimoresi/vapper_multi_projects)

I have 3 folders inside of src: 2 "project_\*" and 1 "shared"
What i need is the 2 project_* to use assets located in the "shared" dir.
I updated my package.json scripts like this:
```js
"scripts": {
    "dev:project_composition": "vapper dev src/project_composition/main.js",
    "build:project_composition": "vapper build --dest dist_project_composition src/project_composition/main.js",
    "start": "vapper start" //i really don't know how it could be
  }
```
while with the standard vue scripts it should work, it doesn't with the vapper scripts, or I am missing how to set up the scripts.
Basically it try to load modules from "project_*/node_modules" instead of "/node_modules"
Can you help me?
Thank you
